solidityContracts/smartcontract.md

```diff
@@ -0,0 +1,23 @@
+# Documentation for using smartcontract functionality:
+
+## Steps to follow:
+
+1. Compile and check the smart contract before placing it in the folder solidityContract.
```

Comment:
May we have a `makefile` that compiles the code so that the unaware users of Solidty can still benefit from it? e.g., something like the idea shared here: https://github.com/trufflesuite/truffle/issues/2442

Reply:
The current commands compile the contract before converting them to bin files